otel/README.md

```diff
@@ -95,3 +98,4 @@ Need help? Contact [Datadog support][6].
 [4]: https://github.com/open-telemetry/opentelemetry-collector/tree/master/receiver/hostmetricsreceiver
 [5]: https://github.com/DataDog/integrations-core/blob/master/opentelemetry/metadata.csv
 [6]: https://docs.datadoghq.com/help/
+[7]: https://docs.datadoghq.com/tracing/setup_overview/open_standards/#datadog-agent-otlp-ingestion-support
```

Comment:
Can you check this link? I don't see the `datadog-agent-otlp-ingestion-support` anchor on the page.